src/windows7.rs

```diff
@@ -0,0 +1,37 @@
+//! Implementation for Windows 7 and 8
```

Comment:
This is misleading, because it for Windows 7 and later, and in some ways even Windows XP and later, not just 7 and 8.

Reply:
Fixed to note that it works for XP and later